help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,168 @@
+|phone_sha256|Phone numbers hashed with the SHA256 algorithm|Both plain text and SHA256 hashed phone numbers are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|
+|email_lc_sha256|Email addresses hashed with the SHA256 algorithm|Both plain text and SHA256 hashed email addresses are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|
+|extern_id|Custom user IDs|Select this target identity when your source identity is a custom namespace.|
+
+{style="table-layout:auto"}
+
+## Supported audiences
+
+This section describes which type of audiences you can export to this destination.
+
+|**Audience**|**Supported**|**Description**|**origin**|                      
+|---|---|---|---|
+|Segmentation Service|✓|Audiences generated through the Experience Platform [[Segmentation Service]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home).|
+|Custom uploads|x|Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into Experience Platform from CSV files.|
+
+{style="table-layout:auto"}
+
+## Export type and frequency
+
+Refer to the table below for information about the destination export type and frequency.
+|**Audience**|**Supported**|**Description origin**|            
+|---|---|---|      
+|Segmentation Service|✓|Audiences generated through the Experience Platform [[Segmentation Service]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home).|
+Custom uploads|X|Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into Experience Platform from CSV files.
+
+{style="table-layout:auto"}
+
+## Connect to the destination
+
+>[!IMPORTANT]
+>To connect to the destination, you need the **View Destinations** and **Manage and Activate Dataset Destinations** [[access control permissions]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/home#permissions). Read the [[access control overview]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/access-control/ui/overview) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [[destination configuration tutorial]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/connect-destination). In the destination configuration workflow, fill in the fields listed in the two sections below.
```

Comment:
this formatting for linking is incorrect, please follow the one from the doc template